src/main/proto/wfa/measurement/reporting/v2alpha/metric.proto

```diff
@@ -128,47 +128,66 @@ message MetricSpec {
   VidSamplingInterval vid_sampling_interval = 5;
 }
 
+// Statistics of a scalar value
+message UnivariateStatistics {
+  // The standard deviation of the associated scalar variable value.
+  //
+  // Can be used to find different confidence intervals.
+  double standard_deviation = 1;
+}
+
 // The calculation result of a metric
 message MetricResult {
```

Comment:
Regarding the naming, do you think MetricValue or MetricValues (and hereinafter) is better? As it is very common to say calculating the value of a metric, or metric value, or calculation result. But I do not have a strong preference. You can decide it.